lib/AppInfo/Application.php

```diff
@@ -77,6 +80,7 @@ public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(NodeRemovedFromCache::class, FileListener::class);
 		$context->registerEventListener(NodeWrittenEvent::class, FileListener::class);
 		$context->registerEventListener(AppDisableEvent::class, AppDisableListener::class);
+		$context->registerEventListener(UserDeletedEvent::class, UserDeletedListener::class);
```

Comment:
Perhaps we also need Group member addedf/removed?

Reply:
Good point

Reply:
how about adding it later with complete group based ACLs. We won't need to translate group -> users then, only the list of groups the user is part of, on the fly when querying.
In the backend it would be checked if a file can be accessed by the user or one of the groups the user is part of.
We can maintain a local list in the backend for user-group mappings but fetching it on the fly is not expensive, so we won't even need the listener if a user was added/removed from a group.

Reply:
mmh. I'd stick with user-based ACLs for this iteration, I think.